packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -515,6 +531,18 @@ class Scaffold extends StatefulWidget {
     }
   }
 
+  static bool hasEndDrawer(BuildContext context, { bool registerForUpdates: true }) {
```

Comment:
Does this function have any callers?

Reply:
Yes, it has invocation in app_bar.dart

Reply:
Sorry, I'm missing something.  Can you tell me the line number?  Line 340 calls ScaffoldState.hasEndDrawer, which is on line 565 of this file rather than here.

Reply:
^^^

Reply:
Apologies. 
The function would have a similar applicability as in case of `hasDrawer`, just for the endDrawer

Reply:
That function appears to have been added in 83a4cf269f7795c6d9159fa7521a738b19d992d8 and was never used.  Unless there's a specific reason to add this function, I'd skip it (and the associated machinery in _ScaffoldScope.

Reply:
Sure. I'll skip it.